src/Data/Integer/IntConstruction.agda

```diff
@@ -0,0 +1,77 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Construction of integers as a pair of naturals
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --cubical-compatible #-}
+
+module Data.Integer.IntConstruction where
+
+open import Data.Nat.Base as ℕ using (ℕ)
+open import Function.Base using (flip)
+open import Relation.Binary
+open import Relation.Binary.PropositionalEquality
+
+infixl 6 _⊖_
+
+record ℤ : Set where
+  constructor _⊖_
+  field
+    minuend : ℕ
+    subtrahend : ℕ
+
+infix 4 _≃_ _≤_ _≥_ _<_ _>_
+
+record _≃_ (i j : ℤ) : Set where
+  constructor mk≃
+  field
+    eq : ℤ.minuend i ℕ.+ ℤ.subtrahend j ≡ ℤ.minuend j ℕ.+ ℤ.subtrahend i
```

Comment:
First observation from `Intproperties`: use of `sym` to flip rewrites in the RHS of such equations... so suggest instead
```suggestion
    eq : ℤ.minuend i ℕ.+ ℤ.subtrahend j ≡ ℤ.subtrahend i ℕ.+ ℤ.minuend j
```
?

Reply:
The reason I did it this way was so that reflexivity is exactly `refl`. But you're right, that doesn't seem to be worth the tradeoff